runner/src/server/services/uploadService.ts

```diff
@@ -60,25 +60,28 @@ export class UploadService {
     return this.parsedDocumentUploadResponse(res);
   }
 
-  parsedDocumentUploadResponse(res: http.IncomingMessage) {
-    let error: string | undefined;
+  parsedDocumentUploadResponse(res: http.IncomingMessage, error?: any) {
     let location: string | undefined;
```

Comment:
Typically you don't want to mutate a function parameter. I think technically it's fine here because when you pass in a primitive to functions in js, they will copy them instead of using a reference pointer. looks like we're just passing in a string here.

Worst case scenario, you pass in an object, in this case `error` as a reference pointer. If any other function is using the same error at the same time, you'll end up reassigning and end up with unintended side effects

```.ts
// set default
let parsedError: string = "There was an error uploading your file";

if (error) {
  // parse
} else {
  switch (res.statusCode) {
   case 413: 
     parsedError = '..';
  }
}

```